package/libs/libpcap/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libpcap
 PKG_VERSION:=1.9.1
-PKG_RELEASE:=3
+PKG_RELEASE:=4
```

Comment:
I prefer 3.1 here, so it's clear it's not a 1.9.1-4 taken directly from master.

Reply:
ok, sounds plausible. I have changed it.